.changeset/lucky-years-doubt.md

```diff
@@ -0,0 +1,96 @@
+---
+'@twilio-paste/alert': minor
+'@twilio-paste/alert-dialog': minor
+'@twilio-paste/anchor': minor
+'@twilio-paste/avatar': minor
+'@twilio-paste/badge': minor
+'@twilio-paste/base-radio-checkbox': minor
+'@twilio-paste/breadcrumb': minor
+'@twilio-paste/button': minor
+'@twilio-paste/button-group': minor
+'@twilio-paste/callout': minor
+'@twilio-paste/card': minor
+'@twilio-paste/chat-composer': minor
+'@twilio-paste/chat-log': minor
+'@twilio-paste/checkbox': minor
+'@twilio-paste/code-block': minor
+'@twilio-paste/combobox': minor
+'@twilio-paste/data-grid': minor
+'@twilio-paste/date-picker': minor
+'@twilio-paste/description-list': minor
+'@twilio-paste/detail-text': minor
+'@twilio-paste/disclosure': minor
+'@twilio-paste/display-heading': minor
+'@twilio-paste/display-pill-group': minor
+'@twilio-paste/file-picker': minor
+'@twilio-paste/file-uploader': minor
+'@twilio-paste/form': minor
+'@twilio-paste/form-pill-group': minor
+'@twilio-paste/heading': minor
+'@twilio-paste/help-text': minor
+'@twilio-paste/in-page-navigation': minor
+'@twilio-paste/inline-code': minor
+'@twilio-paste/inline-control-group': minor
+'@twilio-paste/input': minor
+'@twilio-paste/input-box': minor
+'@twilio-paste/label': minor
+'@twilio-paste/list': minor
+'@twilio-paste/menu': minor
+'@twilio-paste/minimizable-dialog': minor
+'@twilio-paste/modal': minor
+'@twilio-paste/pagination': minor
+'@twilio-paste/paragraph': minor
+'@twilio-paste/popover': minor
+'@twilio-paste/radio-button-group': minor
+'@twilio-paste/radio-group': minor
+'@twilio-paste/screen-reader-only': minor
+'@twilio-paste/select': minor
+'@twilio-paste/separator': minor
+'@twilio-paste/side-modal': minor
+'@twilio-paste/skeleton-loader': minor
+'@twilio-paste/spinner': minor
+'@twilio-paste/switch': minor
+'@twilio-paste/table': minor
+'@twilio-paste/tabs': minor
+'@twilio-paste/textarea': minor
+'@twilio-paste/time-picker': minor
+'@twilio-paste/toast': minor
+'@twilio-paste/tooltip': minor
+'@twilio-paste/truncate': minor
+'@twilio-paste/core': minor
```

Comment:
Let's make this a major for all

Reply:
OK, I won't bother amending my changeset. Let's bundle in a few majors together